setup.py

```diff
@@ -42,11 +42,12 @@
     python_requires='>=3.6',
     install_requires=['numpy>=1.21',
                       'rioxarray>=0.4',
-                      'pdal>=2.2',
+                      'pdal>=2.2'
                       'geopandas>=0.9',
                       'shapely>=1.7',
                       'scipy>=1.6',
-                      'pathlib>=1.0'],
+                      'requests>=2.25',
+                      'boto3>=1.17'],
     extras_require={
         'dev': ['check-manifest'],
     },
```

Comment:
Now that GH actions is using Conda, these changes and the ones in `requirements.txt` can be kept or reverted. If they are fixing/improving, then no harm in leaving them here. This could be useful if GeoFabrics is released to PYPI in the future.